tests/nested/manual/hybrid-remodel/prepare-device

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# 10.0.2.2 is the host from a nested VM
+snapctl set device-service.url=http://10.0.2.2:11029
```

Comment:
I wonder if this is really needed in hybrid, I will test without it.

Reply:
In another tests were there is serial authority generic, I still need to have this. Otherwise it fails because the model is signed by developer1.